smart-contracts/data/ibc_mapping.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2":"ATOM",
```

Comment:
is this something we can replace with a call like  
`sifnoded q ibc-transfer denom-trace D87BC708A791246AA683D514C273736F07579CBD56C9CA79B7823F9A01C16270 --node=https://rpc.sifchain.finance:443`